sdk/storage/storage-queue/src/Pipeline.ts

```diff
@@ -177,10 +201,15 @@ export function newPipeline(
     new BrowserPolicyFactory(),
     deserializationPolicy(), // Default deserializationPolicy is provided by protocol layer
     new RetryPolicyFactory(pipelineOptions.retryOptions),
-    new LoggingPolicyFactory(),
-    credential
+    new LoggingPolicyFactory()
   ];
 
+  if (isNode) {
+    // ProxyPolicy is only avaiable in Node.js runtime, not in browsers
+    factories.push(proxyPolicy(getDefaultProxySettings((pipelineOptions.proxy || {}).url)));
```

Comment:
>(pipelineOptions.proxy || {}).url [](start = 55, length = 33)

`(pipelineOptions.proxy || {}).url`

looks not right, `{}.url` is undefined

Reply:
Updated this to `getDefaultProxySettings(pipelineOptions.proxy ? pipelineOptions.proxy.url : undefined)`

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/4041